### PR TITLE
chore: add enabled property to deploy tests

### PR DESCRIPTION
**Pull request by hectorj2f**
We were deploying the test resources, that should not happen on production.

### CODE REVIEW

File: staging/traefik/templates/tests/test-configmap.yaml [hunk 1]
```diff
@@ -12,4 +13,4 @@ data:
     @test "Test Access" {
       curl -D - http://{{ template "traefik.fullname" . }}/
     }
-
+{{end}}
```
Comment (samvantran) [inline staging/traefik/templates/tests/test-configmap.yaml]: This doesn't need to be `{{- end}}`? You started the `{{- if` that way

File: staging/traefik/templates/tests/test.yaml [hunk 2]
```diff
@@ -40,3 +41,4 @@ spec:
   - name: tools
     emptyDir: {}
   restartPolicy: Never
+{{end}}
```
Comment (samvantran) [inline staging/traefik/templates/tests/test.yaml]: same here

Comment (hectorj2f) [reply staging/traefik/templates/tests/test.yaml]: thx good catch ;)

File: staging/traefik/templates/tests/test-configmap.yaml (hunk 1 above)
Comment (hectorj2f) [reply staging/traefik/templates/tests/test-configmap.yaml]: thx good catch ;)

**Pull request merged**